app/models/journeys/additional_payments_for_teaching.rb

```diff
@@ -9,5 +9,11 @@ module AdditionalPaymentsForTeaching
     VIEW_PATH = "additional_payments"
     I18N_NAMESPACE = "additional_payments"
     POLICIES = [Policies::EarlyCareerPayments, Policies::LevellingUpPremiumPayments]
+
+    def forms
+      {
+        "supply-teacher" => SupplyTeacherForm
```

Comment:
Suggestion: again, going with Rails' 'convention over configuration' principle perhaps the form class name could be inferred from the slug name.